xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd

```diff
@@ -250,7 +250,10 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="TariffRef" minOccurs="0"/>
 			<xsd:element ref="FareStructureElementRef" minOccurs="0"/>
 			<xsd:element ref="FareElementInSequenceRef" minOccurs="0"/>
-			<xsd:element ref="DistanceMatrixElementRef" minOccurs="0"/>
+			<xsd:choice  minOccurs="0">
```

Comment:
since both choices are in the same substitution group, I'm not sure the use of choice is really effective... but it does not harm anyway